backend/app/tests/api/routes/test_assistants.py

```
@@ -29,3 +51,277 @@ def test_ingest_assistant_success(
+        headers=normal_user_api_key_header,
+    )
+
+    assert response.status_code == 400
+    response_data = response.json()
+    assert response_data["error"] == "Vector store ID vs_test_999 not found in OpenAI."
+
+
+def test_update_assistant_success(
+    client: TestClient,
+    db: Session,
+    normal_user_api_key_header: dict,
```

Ruff (0.12.2)

Unused function argument: `normal_user_api_key_header`

(ARG001)